next.js/packages/one/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@nextjs-example/one",
+  "private": true,
+  "main": "src/main.js",
+  "dependencies": {
+    "@types/is-odd": "3.0.1",
```

Comment:
nit: we should set a good example by not overconstraining. You probably don't need any constraint on the version of this package, e.g. `*`

Reply:
👍 I'll set it to `3.x.x` as good practice to constrain the major at least